data/data/alibabacloud/cluster/master/main.tf

```diff
@@ -0,0 +1,48 @@
+locals {
+  description = "Created By OpenShift Installer"
+  prefix      = var.cluster_id
+}
+
+data "alicloud_instances" "master_data" {
+  ids = alicloud_instance.master.*.id
+}
+
+resource "alicloud_instance" "master" {
+  count             = length(var.vswitch_ids)
+  resource_group_id = var.resource_group_id
+
+  instance_name              = "${local.prefix}_master_${count.index}"
```

Comment:
Please add the additional field for `host_name` https://registry.terraform.io/providers/aliyun/alicloud/latest/docs/resources/instance#host_name

Reply:
Ok, I have updated it.

Reply:
Please match the hostname above.